### PR TITLE
LPS-167635 Redirect pattern validation

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Issue https://issues.liferay.com/browse/LPS-167635

I replicated the same validation we have in the Alias section for the destination input. The only difference is that the input is not required (because it could be left as empty and backend uses this to remove the first row, or ignore the value if it is in another row)

Screenshot 2022-11-04 at 15 11 02

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]
This solution do not allow the introduction of destination URLs the previously accepted friendlyURLs

image

File: modules/apps/redirect/redirect-web/src/main/resources/META-INF/resources/js/RedirectPatterns.js [hunk 1]
```diff
@@ -95,6 +114,29 @@ const PatternField = ({
...
+
+							{Liferay.Language.get('this-url-is-not-supported')}
+						</ClayForm.FeedbackItem>
+
+						<div
+							className="small"
+							dangerouslySetInnerHTML={{
+								__html: sub(
+									Liferay.Language.get(
+										'enter-an-absolute-url'
```
Comment (AliciaGarciaGarcia) [inline modules/apps/redirect/redirect-web/src/main/resources/META-INF/resources/js/RedirectPatterns.js:131]: We support friendlyURLs too

**Comment by AliciaGarciaGarcia** [comment]
I added a commit to allow relative urls

image

But it keep letting save the configuration.

This should be done on back or there is a way to do it on react?

// @boton @ambrinchaudhary

**Comment by ambrinchaudhary** [comment]
> This solution do not allow the introduction of destination URLs the previously accepted friendlyURLs
> 
> image

Hey @AliciaGarciaGarcia , I'm a bit confused with this. Should we also support friendly-urls in the other section, when creating an alias/redirection? or it is only for patterns? 

Also, now the second line of the help message does not make sense. Should we provide a more usefull message (I guess), or just remove the second line?

Screenshot 2022-11-08 at 18 10 55

**Comment by AliciaGarciaGarcia** [comment]
Changed the message
image

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by marcogalluzzi** [comment]
ci:forward

**Pull request closed without merging**